### DISCUSSION TITLE
Community management with Gov4Git built on go-git

### DISCUSSION THREAD

**Discussion by petar** [go-git/go-git#1053] [General]
Hi folks!

I am writing on behalf of the [Gov4Git](https://github.com/gov4git/gov4git) project, which is entirely built on top of go-git.

We thought we would extend our first invitation to you, because you are our main dependency. We want to invite your team to try out our software, essentially take it for a "test drive" and let us know what you think! 

You would have our full support in this process. Ultimately, our success depends on yours.

Gov4Git is a protocol for managing open source collaboration on GitHub. It addresses a slew of common frictions and inefficiencies in collaborative open-source projects. 

Gov4Git will enable you to:
- __prioritize issues__ in a transparent and fair community-driven manner,
- __relieve maintainers__ from being a bottleneck in triage,
- objectively __quantify individual contributions__ based on peer review mechanisms, and even
- make __decision-making__ on contentious issues inclusive, open, and democratic.

The relevant links for acquainting yourselves with Gov4Git are:
- [Our website](https://gov4git.org)
- [Overview of the product and how it integrates into a GitHub project](https://www.gov4git.org/s/DEPLOY-MANAGE-COLLABORATE-OVERVIEW.pdf)
- [Detailed explanation of our governance mechanism for open-source communities](https://medium.com/@gov4git/waimea-collective-management-and-compensation-for-open-source-communities-affa9b9093e9)

We truly thank you for your time and consideration.

**Comment by pjbgf** [comment]
Hey @petar,

I moved this into a discussion as to invite others to also share their opinions as well.

Overall, I do like the idea behind gov4git and I think it could be very useful to gauge what issues are most valued to the community.

1. Does it take into account current indicators (# of :+1: on issues) for the initial stats?
2. Are there any plans to have it hosted somewhere so folks don't need to run it on their machines? On my specific workflow, I would rather lean on a website than running an application on my machine.

**Reply by petar** [reply]
Hi @pjbgf. 

1. The software doesn't currently take into account existing thumbs up. This is a feature that could be included. 

2. The governance system does not run on users' computers. It runs in a GitHub Action inside a dedicated repository, created for keeping track of governance records. 

Let me know if you have other questions.

**Reply by petar** [reply]
Also, you can retroactively drop governance credits on existing issues to reflect their current thumbs up counts. If this is a show stopper, we could add this feature.
